docs/user/FlowVariables.md

```diff
@@ -33,9 +33,9 @@ public platforms supported by the OpenROAD flow.
 
 
 Note:
--   = indicates default definition assigned by the tool
--   ?= indicates that the variable value may be reassigned with design `config.mk`
--   N/A indicates that the variable/files is not supported currently.
+-   ☑️ indicates default definition assigned by the tool
+-   ✔️ indicates that the variable value may be reassigned with design `config.mk`
+-   ❌indicates that the variable/files is not supported currently.

```

Comment:
I think we should keep `=` and `?=` as they correlate with the syntax in `.mk` files. Variables that are not supported should not be documented as there's no benefit.

Reply:
Some of the variables are supported for a few/ only one platform - should we keep them?

Reply:
Yes, but in an isolated section. Here we should keep variables that apply to all. We can add them as a subsection under "Optional Variables" at the end of the document.